en/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Privacy Policy for ruby-lang.org"
+lang: en
+---
+
+This privacy policy covers ruby-lang.org.
+
+## Email
+
+We will not give away your email address to anyone, who is not related to the operations of ruby-lang.org. We will also never ask you to send us any of your passwords via email.
+
+## Logfiles
+
+ruby-lang.org record access logs of the requests that reach the web servers, but we use those files for issue investigation and statistical purposes.
```

Comment:
As w.r-l.o uses GitHub Pages, it's worth to mention https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection

> www.ruby-lang.org uses GitHub Pages in its backend thus it is subject to GitHub's Data Collection. Refer to https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection for GitHub's privacy statement.